src/runtimes.rs

```diff
@@ -178,6 +198,15 @@ impl Staking for DefaultNodeRuntime {}
 impl Runtime for DefaultNodeRuntime {
     type Signature = MultiSignature;
     type Extra = DefaultExtra<Self>;
+
+    fn register_type_sizes(event_type_registry: &mut EventTypeRegistry<Self>) {
+        event_type_registry.with_system();
+        event_type_registry.with_balances();
+        event_type_registry.with_session();
+        event_type_registry.with_contracts();
+        event_type_registry.with_sudo();
+        register_default_type_sizes(event_type_registry);
```

Comment:
Note that this implementation must be amended manually for new and removed pallets. It would be possible to rework the macros to do this but since this is a short term solution I think this is good enough.